xml/containers-get.xml

```diff
@@ -1,562 +1,635 @@
-       </entry>
-       <entry align="left" valign="top">
-        <para>
-         org.opencontainers.image.<emphasis role="strong">version</emphasis>
-         com.suse.sle.base.<emphasis role="strong">version</emphasis>
-        </para>
-       </entry>
-       <entry align="left" valign="top">
-        <para>
-         org.opencontainers.image.version
-        </para>
-       </entry>
-       <entry align="left" valign="top">
-        <para>
-         Image version
-        </para>
-       </entry>
-       <entry align="left" valign="top">
-        <para>
-         &lt;codestream&gt;.&lt;sp&gt;.&lt;cicnt&gt;.&lt;bldcnt&gt; 15.0.4.2
-        </para>
-       </entry>
-      </row>
-      <row>
-       <entry align="left" valign="top">
-        <para>
-         Yes
-        </para>
-       </entry>
-       <entry align="left" valign="top">
-        <para>
-         org.opencontainers.image.<emphasis role="strong">created</emphasis>
-         com.suse.sle.base.<emphasis role="strong">created</emphasis>
-        </para>
-       </entry>
-       <entry align="left" valign="top">
-        <para>
-         org.opencontainers.image.created
-        </para>
-       </entry>
-       <entry align="left" valign="top">
-        <para>
-         Timestamp of image build
-        </para>
-       </entry>
-       <entry align="left" valign="top">
-        <para>
-         2018-07-27T14:12:30Z
-        </para>
-       </entry>
-      </row>
-      <row>
-       <entry align="left" valign="top">
-        <para>
-         No
-        </para>
-       </entry>
-       <entry align="left" valign="top">
-        <para>
-         org.opencontainers.image.<emphasis role="strong">vendor</emphasis>
-         com.suse.sle.base.<emphasis role="strong">vendor</emphasis>
-        </para>
-       </entry>
-       <entry align="left" valign="top">
-        <para>
-         org.opencontainers.image.vendor
-        </para>
-       </entry>
-       <entry align="left" valign="top">
-        <para>
-         Image vendor
-        </para>
-       </entry>
-       <entry align="left" valign="top">
-        <para>
-         SUSE LLC
-        </para>
-       </entry>
-      </row>
-      <row>
-       <entry align="left" valign="top">
-        <para>
-         No
-        </para>
-       </entry>
-       <entry align="left" valign="top">
-        <para>
-         org.opencontainers.image.<emphasis role="strong">url</emphasis>
-         com.suse.sle.base.<emphasis role="strong">url</emphasis>
-        </para>
-       </entry>
-       <entry align="left" valign="top">
-        <para>
-         org.opencontainers.image.url
-        </para>
-       </entry>
-       <entry align="left" valign="top">
-        <para>
-         Further information
         </para>
-       </entry>
-       <entry align="left" valign="top">
         <para>
-         <link xlink:href="https://www.suse.com/products/server/" />
+          The opposite is true for &opensuse; base images, where labels are not
```

Comment:
Maybe "not" should be emphasized here?

Reply:
It's within a note, it uses the word "opposite" in the same sentence, so I guessed that was emphasis enough. :)